src/CommandingOfficers/AWBW/AWBWCommander.java

```diff
@@ -0,0 +1,71 @@
+package CommandingOfficers.AWBW;
+
+import java.util.ArrayList;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderAbility;
+import CommandingOfficers.CommanderInfo;
+import CommandingOfficers.CommanderInfo.InfoPage;
+import Engine.GameScenario;
+import Engine.UnitMods.UnitDamageModifier;
+import Engine.UnitMods.UnitModifier;
+import Terrain.MapMaster;
+import Units.Unit;
+import Units.UnitDelta;
+
+public abstract class AWBWCommander extends Commander
+{
+  private static final long serialVersionUID = 1L;
+  public static final InfoPage AWBW_MECHANICS_BLURB = new InfoPage(
+      "Gain +10 attack and defense on activating a power.\n"
+    + "CO powers never charge while a power is active.\n"
+    + "COP charge gained from combat is funds damage taken and half of funds damage dealt.\n"
+    + "On each power activation, each star in your meter costs 1/5 more of the base star cost.\n"
+    + "This continues for 10 activations, at which point the star cost settles at triple the base star cost.\n"
+    );
+
+  public AWBWCommander(CommanderInfo info, GameScenario.GameRules rules)
+  {
+    super(info, rules);
+    // Sauce: https://docs.google.com/document/d/e/2PACX-1vQUyQIFa3sLfKU3W1ijlXQ7s_453XaJCiLle9JeBAvkbkYGcIXqVsXnlNs9xpHktTIDpEEH8xG9e9W0/pub
+    // Repairing does not round up partial HP. (Note: Joining does round up partial HP)
+    //     Exception: if the heal can overheal, then the chip damage will be healed for free.
+    //     (8.1HP unit on city will heal to 10HP, costing 1HP of repair).
+    roundUpRepairs = false;
+  }
+
+  @Override
+  public int calculateCombatCharge(UnitDelta minion, UnitDelta enemy, boolean isCounter)
+  {
+    if( null != getActiveAbility() )
+      return 0;
+    // TODO: Copy/paste super's code and delete the HP-based bit once the parent PR settles down.
```

Comment:
Does this still have a parent PR? Or does the child PR address this?

Reply:
Looks like it came from Trilogy, which has addressed it already. So we can probably just delete the function here? And the only reason Trilogy needed to reimplement it is because our Commander is nonconformant to the carts, because it counts both HP and funds-based charge accruals, right?

Reply:
And they don't all agree with how much it takes to make a star, either.

Whatever, DefendPeace COs OP.